dashboard/app/views/levels/editors/fields/_browser_tts.html.haml

```diff
@@ -0,0 +1,7 @@
+%h1.control-legend{data: {toggle: "collapse", target: "#browser_tts_area"}}
+  Browser Text-to-Speech
+
+#browser_tts_area.in.collapse
+  %p Enables native browser text-to-speech on all enabled components, such as instructions and validations. Currently only available in Lab2 labs.
```

Comment:
Nit: This is levelbuilder user-facing text, correct? If so, is it important to indicate that it's only for Lab2 labs? I feel like it's more important for engineers to understand that limitation, so I wonder if it'd be better as a simple comment rather than rendered HTML.

Reply:
Yeah, that's fair - this checkbox is only ever included on Lab2 level edit pages (panels and music so far) so maybe not necessarily to call out for levelbuilders.